File: app/src/main/java/com/ve/lawyer/activity/SplashACtivity.java
package com.ve.lawyer.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import com.ve.lawyer.R;

public class SplashACtivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_a_ctivity);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent mainIntent = new Intent(SplashACtivity.this, UserTypeActivity.class);
                startActivity(mainIntent);
                finish();
            }
        }, 3000);
    }
}

File: app/src/main/java/com/ve/lawyer/ui/common/SplashActivity.kt
package com.ve.lawyer.ui.common

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.ve.lawyer.R
import com.ve.lawyer.utils.SlideAnimationUtil

class SplashActivity : AppCompatActivity() {

    private var DELAY_TIME : Long =  1500

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        Handler().postDelayed({
//            if (!Prefs.getString(AppConstant.USER_ID).isEmpty()) {
//                startActivity(Intent(this@SplashActivity, HomeActivity::class.java))
//            } else {
//                startActivity(Intent(this@SplashActivity, LoginActivity::class.java))
//            }
            startActivity(Intent(this@SplashActivity, UserTypeSelectionActivity::class.java))
            finish()
            SlideAnimationUtil.slideNextAnimation(this@SplashActivity)
        },DELAY_TIME)
    }
}